EventBrite/Models/EventBriteItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Etch.OrchardCore.Fields.EventBrite.Models.Dto;
+using System;
+
+namespace Etch.OrchardCore.Fields.EventBrite.Models
+{
+    public class EventBriteItem
+    {
+        public EventBriteItem()
```

Comment:
If this isn't used anywhere, can we remove it?

Reply:
This has a reference in the `EditAsync` for if we somehow have a field without any data.